### PR TITLE
Add docs for partials that need to be provided

### PR DESCRIPTION
**Pull request by skukx**
**Description**

Due to !2513 we need to provide a partial for the api to render a
payment method.

**Checklist:**
- [x] I have followed [Pull Request guidelines](https://github.com/solidusio/solidus/blob/master/CONTRIBUTING.md#pull-request-guidelines)
- [x] I have added a detailed description into each commit message
- [ ] I have updated Guides and README accordingly to this change (if needed)
- [ ] I have added tests to cover this change (if needed)

### CODE REVIEW

File: core/app/models/spree/payment_method.rb [hunk 1]
```diff
@@ -199,6 +199,9 @@ def display_on
     #     4. app/views/spree/admin/payments/source_views/_{partial_name}.html.erb
     #     The view that represents your payment method on orders in the backend
     #
+    #     5. app/views/spree/api/payments/source_views/_{partial_name}.json.jbuilder
+    #     The view that represents your payment method on orders in the backend
```
Comment (kennyadsl) [inline core/app/models/spree/payment_method.rb]: `in the backend` ➡️ `in the api`?

Comment (skukx) [reply core/app/models/spree/payment_method.rb]: Good Catch!

**Comment by kennyadsl** [comment]
@skukx not sure what's happened here, can you please restart the build in your CircleCI account?

**Comment by skukx** [comment]
I wonder if the force push broke the integration with circle

**Pull request merged**